fm.reaper.Reaper.yml

```diff
@@ -0,0 +1,62 @@
+app-id: fm.reaper.Reaper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+tags:
+ - proprietary
+command: /app/REAPER/reaper
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  - --device=all
+  - --filesystem=home
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+  - --system-talk-name=org.freedesktop.Flatpak
+  - --env=ALSA_CONFIG_PATH=
+  - --env=LV2_PATH=/app/extensions/Plugins/lv2
+  - --env=DSSI_PATH=/app/extensions/Plugins/dssi
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --env=VST_PATH=/app/extensions/Plugins/vst
+  - --env=VST3_PATH=$HOME/.vst3:/app/extensions/Plugins/vst3
```

Comment:
note: this won't work. $HOME is not substituted. The trick is to have a wrapper script and set it there. Note that in theory a VST3 host should always look into `$HOME/.vst3` so I'm not sure if this is necessary.

 LV2 need the same treatment.